src/Autocomplete/src/Form/WrappedEntityTypeAutocompleter.php

```diff
@@ -89,7 +89,7 @@ public function getLabel(object $entity): string
             return $this->propertyAccessor->getValue($entity, $choiceLabel);
         }
 
-        if ($choiceLabel instanceof ChoiceLabel) {
+        if ($choiceLabel instanceof Cache\ChoiceLabel) {
```

Comment:
A particular reason for this change ?

Reply:
To have only one use.

Reply:
I'm not sure this offers a better readability, but let's see what others think

Reply:
> I'm not sure this offers a better readability, but let's see what others think

Used in https://github.com/symfony/symfony/blob/0d9562ff6cdda11c71f0eb2bce0076f0d3a8ea9f/src/Symfony/Component/Form/ChoiceList/Factory/CachingFactoryDecorator.php#L89